webapi/src/main/scala/org/knora/webapi/core/AppServer.scala

```diff
@@ -181,7 +181,7 @@ object AppServer {
   /**
    * The live AppServer
    */
-  val live: ZIO[AppServerEnvironment, Nothing, Unit] =
+  val make: ZIO[AppServerEnvironment, Nothing, Unit] =
```

Comment:
What was the reason to change this name?

Reply:
Often we used `live` as something which returns a `ZLayer`, which is not the case here as this produces a `ZIO`. 

The renaming also aligns with the same in `InstrumentationServer` so that the startup is 
`InstrumentationServer.make *> AppServer.make *> ZIO.never`
rather than the more chaotic
`InstrumentationServer.make *> AppServer.live *> ZIO.never`.

Reply:
makes sense